api/src/utils/default-user.ts

```diff
@@ -0,0 +1,55 @@
+// TODO: audit this object to find out which properties need to be updated.
+export const defaultUser = {
```

Comment:
```suggestion
import { user } from "@prisma/client";

export const defaultUser: Omit<user, "id" | "email"> = {
```

I actually ran into something similar in one of my own projects - keeping a default record in sync with the schema. Found that enforcing a type helped!

Reply:
Oooh, nice idea!